js/settings.js

```diff
@@ -27,4 +27,32 @@ jQuery( document ).ready( function() {
 			}
 		}
 	);
+
+	$('.activate-addon').click(function(e) {
+		var $this = $(this);
+		var addon = $this.data('addon');
+		$this.addClass('processing');
+
+		$.ajax({
+			url: ajaxurl,
+			type: 'POST',
+			data: {
+				action: 'toggle_addon',
+				nonce: membersAddons.nonce,
+				addon: addon
+			},
+		})
+		.done(function(response) {
+			$this.find('.action-label').html(response.data.action_label);
+			var svg = $this.find('svg');
+			svg.removeClass();
+			svg.addClass(response.data.status);
+		})
+		.fail(function(response) {
+			console.log(response.data.msg);
```

Comment:
@renventura we'll probably want to remove this before going live. Maybe replace with an alert?

Reply:
@cartpauj Yeah, we can do that.